### PR TITLE
Enable hourly forecast

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -33,11 +33,18 @@ def generate_forecast():
...
+    valid_time_method = ['currently', 'hourly']
+    last_query = place.split(' ')[-1]
+    if last_query in valid_time_method:
+        place = place.replace(last_query, '')  # replace time method(if present) with empty string so that only the place remains. Example query: cebu hourly
+        time_method = last_query
+
     forecast = WeatherForecast()
     coordinates = geocode.get_coordinates()
-    weather_forecast = forecast.generate_forecast(latitude=coordinates['lat'], longitude=coordinates['lng'])
+    weather_forecast = forecast.generate_forecast(latitude=coordinates['lat'], longitude=coordinates['lng'], time_method=time_method)
```
Comment (dashmug) [inline app/views.py]: It's not a strict rule but shorter line lengths are easier to read.

File: app/weather_parser.py [hunk 2]
```diff
@@ -4,8 +4,10 @@
 
 class WeatherForecast:
 
-    def generate_forecast(self, latitude, longitude):
+    def generate_forecast(self, latitude, longitude, time_method):
```
Comment (dashmug) [inline app/weather_parser.py]: I suggest `frequency` as a better parameter name than `time_method`.

**Comment by admiral96** [comment]
Build passed on my fork. Build on main repo naturally fails because this PR needs merging before travis can use the environment variables being set

**Comment by mknecht** [comment]
@admiral96 this does not make sense: this PR's branch is tested on travis. If it adds the env variables it needs, it should succeed.

**Comment by admiral96** [comment]
Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code. 
Ref: https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables

**Comment by mknecht** [comment]
@admiral96 don't make _real_ API calls during tests. if those APIs are unreachable, there is a network problem or your keys run out of access, your tests will fail. That's bad design. Instead mock them with the expected answer.

**Comment by mknecht** [comment]
@admiral96 yeah, like i said, you should not be doing _real_ ones anyway, so you might as well put fake stuff. real credentials should not be committed to no repository.

**Comment by mknecht** [comment]
I love that you worried about this and found a solution, though. Good job @admiral96 :)

**Pull request merged**